ocs_ci/templates/ocs-deployment/flexible-scaling-storage-cluster.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: ocs.openshift.io/v1
+kind: StorageCluster
+metadata:
+  namespace: openshift-storage
+  name: ocs-storagecluster
+spec:
+  flexibleScaling: true
```

Comment:
Only one difference for this file is this `flexibleScaling: true` parameter: 
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/templates/ocs-deployment/storage-cluster.yaml

I don't think we need to do whole storage cluster CR for that, see my other comment what I suggest.

Reply:
The storage_cluster.yaml for flexible scaling also sets the count to 3 and the replica to 1, which is why I created a new file.
Let me know if you would like the changes in the code instead of the new yaml in spite of this.

Reply:
Replied in other comment, let's wait for input from others, but I still prefer to change in the code.